src/Auxiliary/Aux_PauseManually.cpp

```diff
@@ -0,0 +1,54 @@
+#include "GAMER.h"
+
+
+
+
+//-------------------------------------------------------------------------------------------------------
+// Function    :  Aux_PauseManually
+// Description :  Pause a simulation when detecting a file named "PAUSE_GAMER_PAUSE"
+//
+// Note        :  To resume a simulation, simply delete the above file
+//
+// Parameter   :  None
+//
+// Return      :  None
+//-------------------------------------------------------------------------------------------------------
+void Aux_PauseManually()
+{
+
+   const char FileName[] = "PAUSE_GAMER_PAUSE";
+   const int  PauseSec   = 10;   // sleep for PauseSec seconds before checking the target file again
+                                 // --> set rather arbitrarily here
+
+
+// enforce NFS to flush the file information
+   system( "ls > /dev/null" );
+
+
+   bool FirstTime   = true;
```

Comment:
Just curious, what does this variable do for?

Reply:
@ChunYen-Chen It's for printing the following line just once.
```
if ( MPI_Rank == 0  &&  FirstTime )
            Aux_Message( stdout, "\nDetecting file \"PAUSE_GAMER_PAUSE\" --> simulation is paused until this file is deleted ...\n\n" );
```

Reply:
I see. Thanks!